test/e2e/run.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
```

Comment:
do we want to store this file in git? or it's just a temporary testing script?

Reply:
it will be removed